gitpod-io/website
# Document EKS Dual Load Balancer Setup recommended for non CLB users

## Description
This PR introduces a secondary guide that allows those that do not want to or cannot use the AWS Classic load balancer that currently gets created in the AWS Single cluster reference architecture. It instead creates an ALB (for HTTPS traffic) and an NLB (for SSH traffic). The reference archtiecture was also modified to link to said guide in two places: at the beginning upon cluster creation (this can be removed when if we don't need the overridebootstrap command anymore, see https://github.com/gitpod-io/gitpod/issues/11005) and in the ingress section. 

--> Internal [discussion](https://gitpod.slack.com/archives/C01KLC56NP7/p1658923264921369) that lead to this.

## Related Issue(s)

Fixes #

## How to test
-  Look at files changed to ensure I wired things up correctl and created a good transition between the two guides (at least as good as we can make it...)
- Go to netfily preview > docs > self-hosted> installation guides> advanced installation > Setting up your EKS cluster with dual ALB + NLB load balancers
  - Review for correct functionality and logic
  - Review for grammar and spelling

## Release Notes

```release-note
NONE
```


<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2492"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2492"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
